website/api/helpers/merge-json-tables-schema.js

```diff
@@ -0,0 +1,133 @@
+module.exports = {
+
+
+  friendlyName: 'Merge JSON Schema Tables',
```

Comment:
Also, since this returns a value, the convention I'd expect is for the filename (and thus the friendlyName) to begin with "get".  

e.g.
> Get extended osquery tables